Apache-Parquet-Loader/main.py

```diff
@@ -67,14 +66,18 @@ def do_put_arrow_table(flight_client, table_name, arrow_table):
 # Main Function.
 if __name__ == "__main__":
     if len(sys.argv) < 3:
```

Comment:
shouldn't this be `if len(sys.argv) < 4`?

Reply:
Fixed.